speaker_verification/model_evaluation.py

```diff
@@ -2,26 +2,28 @@
 from os.path import join, abspath, dirname
 import csv
 import pathlib
-from random import seed
 
 import numpy as np
-import pytest
 
-from speaker_verification.audio import NUM_FBANKS, NUM_FRAMES, SAMPLE_RATE, read_mfcc, sample_from_mfcc
-from speaker_verification.rescnn_model import DeepSpeakerModel
+from speaker_verification.deep_speaker.audio import (
+    NUM_FRAMES,
+    SAMPLE_RATE,
+    read_mfcc,
+    sample_from_mfcc,
+)
+from speaker_verification.deep_speaker.rescnn_model import DeepSpeakerModel
 
-model_path = join(abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5")
 
-def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv):
+MODEL_PATH = join(
+    abspath(dirname(__file__)), "models", "ResCNN_triplet_training_checkpoint_265.h5"
+)
+
+
+def run_VCSK_Corpus_data(speaker_1, speaker_2, to_csv, dataset_path):
```

Comment:
@baradghimire I've just added the `dataset_path` as a quick fix to the local path.